File: src/judger/tests/data/src_write.cpp
#include<iostream>
#include<fstream>
using namespace std;


int main(int argc, char** argv) {
    int n, a, b;
    cin >> n;
    ofstream fout("my_out.txt");
    fout << "This message should not be successfully written to the file";
    fout.close();
    while (n--) {
        cin >> a >> b;
        cout << a + b << endl;
    }
}

File: celery-queue/judger/tests/data/src_write.py
if __name__ == '__main__':
    n = input()
    with open('my_out.txt', "w") as f:
        f.write("This message should not be successfully written to the file")
    for i in range(int(n)):
        line = input()
        a = line.split()[0]
        b = line.split()[1]
        print(int(a) + int(b))